bundles/InstallBundle/Installer.php

```diff
@@ -319,6 +320,13 @@ public function resolveDbConfig(array $params): array
             $dbConfig['port'] = $params['mysql_port'];
         }
 
+        $mysqlSslCertPath = $params['mysql_ssl_cert_path'];
+        if (!empty($mysqlSslCertPath)) {
+            $dbConfig['driverOptions'] = [
+                PDO::MYSQL_ATTR_SSL_CA => $mysqlSslCertPath
+            ];
+        }
+
```

Comment:
Can be made shorter with no need to create a short-lived variable

```php
        if (!empty($params['mysql_ssl_cert_path'])) {
            $dbConfig['driverOptions'] = [
                PDO::MYSQL_ATTR_SSL_CA => $params['mysql_ssl_cert_path'],
            ];
        }
```

Reply:
@NiklasBr thank you very much for this review! However I have to say that I'll stick to the solution with short-lived variable. Here are my arguments:

- we don't have duplication of array key (no double `mysql_ssl_cert_path` as in your example) so there is smaller possibility to change one key in the future and forgot about second
- above this code there is very similar solution for the `$hostSocketValue` - so we are at least uniform

However I've to agree that probably it the whole installation service could be refactored in the future (to don't have to use array keys but getters e.g.) in this case I'll stick with my solution. But of course I'm open for argumentation :)